inc/db.function.php

```diff
@@ -545,10 +545,10 @@ function exportArrayToDB($TAB) {
 /**
  * Import an array encoded in a simple field in the database
  *
- * @param $DATA data readed in DB to import
+ * @param string $data data readed in DB to import
```

Comment:
Param is uppercase in the code

Reply:
Fixed. Mistake was made as I copied/paste well formed documentation from `DbUtils` class.